### DISCUSSION TITLE
Stuck on the "Learn more about a curated Experience" step

### DISCUSSION THREAD

**Discussion by ArtyomVancyan** [community/community#46488] [GitHub Education] [answered]
I have passed the first few steps of the GitHub Global Campus to-do list and am stuck on the "Learn more about a curated Experience" step. What am I supposed to do to pass this step?

![Screenshot from 2023-02-06 13-55-18](https://user-images.githubusercontent.com/44609997/216944806-01348354-808c-47ab-890c-3fa65fc9b3ec.png)

**Comment by jge162** [comment]
@ArtyomVancyan I believe that is bugged, I have never gotten it to complete to this day. (I gave it like multiple tries on multiple devices), It will not unlock anything so you are fine.

**Reply by ArtyomVancyan** [reply]
Should we not complete those to-dos before the campus ends? Have you asked for support?

**Reply by jge162** [reply]
You probably completed them. They just do not get the Check marks. Idk why. I have gone back and still not working. So you’re fine.

**Reply by ArtyomVancyan** [reply]
I hope you're right :)

**Reply by jge162** [reply]
You do not need hope, I have been on this GitHub since 2017.

**Reply by jge162** [reply]
Those check marks are broken. Look they will never be green LOL
![image](https://user-images.githubusercontent.com/31228460/216954472-84f08427-eaed-4fd5-88f8-a259031d8f08.png)

**Comment by samus-aran** [comment]
Hi everyone! 👋 @ArtyomVancyan & @jge162 

You are able to complete the **"Learn more about a curated Experience"** step by navigating through from the Global Campus dashboard to the experiences pages. Below is a screenshot of where to look on the dashboard for "Curated Experiences with popular offers" !  

![GitHub-Education-Curated_Experiences](https://user-images.githubusercontent.com/4041275/218113469-d6c926f8-a60f-44a4-b6cf-17f7091fa280.png)

This should help you achieve your milestones! 🎉 Let us know when you receive the next milestone 💃

**Reply by ArtyomVancyan** [reply]
Here are the offers I have clicked, and what? The timeline does not move. There is an issue with it.
![Screenshot from 2023-02-10 18-26-02](https://user-images.githubusercontent.com/44609997/218116019-f4dc1dd8-c03c-4124-95cd-d141a1aa3d13.png)

**Reply by samus-aran** [reply]
Hi @ArtyomVancyan.

Did you click on the Curated Experiences themselves and not the **Explore More Offers** button. It should take you to pages that have **/experiences/** in the URL. Look at the earlier screenshot underneath the blue box I have marked out. Click on of those experiences "Intro to Web Dev" as an example.

**Reply by jge162** [reply]
I just tried. It does not complete the milestone. Still remains unchecked.

**Reply by ArtyomVancyan** [reply]
@samus-aran, the same thing happens to me, it does not make sense.

**Reply by samus-aran** [reply]
This is much appreciated responding back so quickly! 😄 

I've highlighted this to the internal team to investigate. When there is an update I'll post it here 👍

**Reply by jge162** [reply]
Thanks. It’s the top two that do not work. 
- explore a student pack offer
- learn more about a curated experience

**Comment by ericdrosado** [comment, marked as answer]
👋 Hi folks! Unfortunately we are in the process of sunsetting the todo list feature for a new exciting feature in the upcoming months. Please expect this list to be removed from your Global Campus dashboard in the short term. 

We apologize for the inconvenience.
